.github/stale.yml

```diff
@@ -0,0 +1,27 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# General configuration
+# Label to use when marking as stale
+staleLabel: stale
+
+
+# Issue specific configuration
+issues:
+  # TODO: Raise the limitPerRun once we are satisfied with the bot's performance
+  limitPerRun: 1
+  daysUntilStale: 14
+  daysUntilClose: 14
+  markComment: >
+    This issue has been automatically marked as stale because it has not had activity in the
+    last 14 days. It will be closed in the next 14 days if no further activity occurs.
+    Thank you for your contributions.
+  closeComment: >
+    This issue has been automatically closed because it has not had activity in the
+    last 28 days. If this issue is still valid, please ping a maintainer and ask them to relabel it.
+    Thank you for your contributions.
+  exemptLabels:
+    - bug
+    - request
+    - request/roadmap
+    - request/contributions-welcome
```

Comment:
Since the plan is to deprecate these labels, we should either 1. Remove them from here and enable stalebot post deprecating the labels, 2. Make a note to remove these two lines at a later date.

Reply:
Will update all those issues and remove the labels.

Reply:
All issues updated, removed the labels and updated this section.